package.json

```diff
@@ -7,6 +7,7 @@
     "build": "ng build",
     "test": "ng test",
     "lint": "ng lint",
+    "pre:e2e": "./node_modules/.bin/webdriver-manager update --versions.chrome 89.0.4389.114",
```

Comment:
Implemented as a temporary workaround due to version mismatch of chromedriver in E2E test.